index.html

```diff
@@ -0,0 +1,179 @@
+                prepare you for your ideal developer job.
+            </p>
+        </div>
+        <div class="title-date">
+            <h3>2021.10.22(FRI) ~ 25(MON)</h3>
+            <p>@ National Technology Center of Turkey</p>
+        </div>
+    </section>
+    <section class="home-program">
+        <div class="program-title">
+            <h4>Main Program</h4>
+            <hr>
+        </div>
+        <ul class="program-list">
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-school"></i>
+                    <h4>Lecture</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-desktop"></i>
+                    <h4>Online Presentation</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div>
+                    <i class="fas fa-certificate"></i>
+                    <h4>Certificate</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-stamp"></i>
+                    <h4>Job Guarantee</h4>
+                </div>
+
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-chalkboard-teacher"></i>
+                    <h4>Strong Team</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+        </ul>
+        <div class="program-paragraph">
+            <a href="#">Join CC Global Summit 2015</a>
+        </div>
+        <div class="program-paragraph-desktop">
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+    <div class="program-speakers-title">
+        <h2>Featured Speakers</h2>
+        <hr>
+    </div>
+
+    <section class="program-speakers">
+
+ 
+       
+
+    </section>
+    <section class="program-speakers">
+        <div class="speakers-more">
+            <button id="loadMore"> More <i class="fas fa-arrow-down"></i></button>
+        </div>
+    </section>
+   
+    <section class="partners">
+        <div class="program-speakers-title">
+            <h2>Partner</h2>
```

Comment:
This section still needs further improvement:
- [ ] Kindly add some **spacing** (padding/ margin) between the _title_ of this section and the _top part of this section's container_, as can be seen in the provided [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 

**Your Partner section:**
![image](https://user-images.githubusercontent.com/12745474/138138983-a351c455-c224-4a06-9202-27a4ee737e8b.png)


**The original Partner section:**
![image](https://user-images.githubusercontent.com/12745474/138139096-e717d43e-ad98-49cf-bd8b-f0aeeb421941.png)